app/assets/stylesheets/hubstats/bootstrap.css

```diff
@@ -654,7 +654,8 @@ blockquote .small:before {
   content: '\2014 \00A0';
```

Comment:
There are a couple changes here that actually do need a deploy thingy so that the deploy page works the same as the pull request page.